extensions/wikia/UserLogin/tests/UserLoginTest.php

```diff
@@ -202,8 +205,7 @@ public function loginDataProvider() {
 		$mockUserParams118 = array(
 			'load' => null,
 			'loadFromDatabase' => null,
-			'checkPassword' => true,
-			'checkTemporaryPassword' => false,
+			'checkPassword' => $passwordSuccess,
 			'params' => array(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "params" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)